ucdlib-directory/src/public/src/elements/ucdlib-directory-research-highlights.tpl.js

```diff
@@ -0,0 +1,33 @@
+import { html } from 'lit';
+
+export function render() {
+  return html`
+    <h2 class="heading--auxiliary">Research Highlights</h2>
+
+    ${this.loading
+      ? html`<p><em>Loading research highlights...</em></p>`
+      : this.error ? html`<p class="research-highlight-alert"> ${this.error}</p>`
```

Comment:
where are your css classes defined? i dont see any rules for `research-highlight-alert`, `research-highlight-url`, etc

Reply:
It is in the /ucdlib-main-website/ucdlib-wp-plugins/ucdlib-assets/assets/css/ucdlib-dev.css.  Is it ok for that to be there?